.github/workflows/release-e2e-workflow-template.yml

```diff
@@ -40,7 +40,7 @@ jobs:
           tar -xzf opensearch-${{ env.VERSION }}-linux-x64.tar.gz
           cd opensearch-${{ env.VERSION }}/
           ./opensearch-tar-install.sh &
-          timeout 900 bash -c 'while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' -u admin:admin -k https://localhost:9200)" != "200" ]]; do sleep 5; done'
+          timeout 1800 bash -c 'while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' -u admin:admin -k https://localhost:9200)" != "200" ]]; do sleep 5; done'
```

Comment:
Looks like both PR changed https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/316/files#r996232962

I am curious if it's necessary

Reply:
Please refer to chat https://amzn-aws.slack.com/archives/C02B92H59SR/p1665781634891699

Reply:
It turns out that bundled OSD 3.0.0 has issues with security. https://github.com/opensearch-project/opensearch-build/issues/2743 Thus this change is not needed.

Reply:
Removed in new PR